### ISSUE TITLE
Gui: blocks: long blocks being visually truncated

### ISSUE THREAD

**Issue by Tom-Willemsen** [ISISComputingGroup/IBEX#3843]
As a user on LARMOR, some of my block names are long, and I do not want these to be truncated in the blocks display in the GUI.

We should look at making the width of the columns dynamic based on block name length.

**Comment by ThomasLohnert** [comment]
This used to be the behaviour, made it fixed size to fix https://github.com/ISISComputingGroup/IBEX/issues/3554. 

How important is this change to them? Is this obscuring information or just a cosmetic thing?

**Comment by Tom-Willemsen** [comment]
Not critical, but it means that they can confuse blocks that have common prefixes. Workaround is available (via tooltip on hovering over the block).

Can we calculate the size once (at config load time) and then not again?

**Comment by ThomasLohnert** [comment]
Yes, can do. Just want to point out this could potentially waste a lot of space as the max column widths would have to be per group, not per column. Resizing columns as blocks move around when resizing the blocks panel is what caused the juddering in the first place.

**Comment by KathrynBaker** [comment]
This is still the case, and is likely a worthwhile change
